### ISSUE TITLE
Job error 'started too many jobs at once'

### ISSUE THREAD

**Issue by mjwweb** [cloudconvert/cloudconvert-php#98]
I've had this script working before, but I'm getting this error now when attempting to convert a file and I'm not sure what's causing it:

<b>Fatal error</b>:  Uncaught CloudConvert\Exceptions\HttpClientException: You have started too many jobs at once in C:\xampp\htdocs\photocloud\vendor\cloudconvert\cloudconvert-php\src\Exceptions\HttpClientException.php:84
Stack trace:
#0 C:\xampp\htdocs\photocloud\vendor\cloudconvert\cloudconvert-php\src\Transport\HttpTransport.php(264): CloudConvert\Exceptions\HttpClientException::paymentRequired(Object(GuzzleHttp\Psr7\Response))
#1 C:\xampp\htdocs\photocloud\vendor\cloudconvert\cloudconvert-php\src\Transport\HttpTransport.php(242): CloudConvert\Transport\HttpTransport-&gt;handleErrors(Object(GuzzleHttp\Psr7\Response))
#2 C:\xampp\htdocs\photocloud\vendor\cloudconvert\cloudconvert-php\src\Transport\HttpTransport.php(158): CloudConvert\Transport\HttpTransport-&gt;sendRequest(Object(GuzzleHttp\Psr7\Request))
#3 C:\xampp\htdocs\photocloud\vendor\cloudconvert\cloudconvert-php\src\Resources\JobsResource.php(47): CloudConvert\Transport\HttpTransport-&gt;post('https://api.clo...', Array)
#4 C:\xampp\htdocs\photocloud\albums\galleriescode\includes\convert.php(36): CloudConvert\Resources\JobsResource-&gt;create(Object(CloudConvert\Models\Job))
#5 {main}
  thrown in <b>C:\xampp\htdocs\photocloud\vendor\cloudconvert\cloudconvert-php\src\Exceptions\HttpClientException.php</b> on line <b>84</b><br />

upload.php:

    require 'vendor/autoload.php';

    use \CloudConvert\CloudConvert;
    use \CloudConvert\Models\Job;
    use \CloudConvert\Models\ImportUploadTask;
    use \CloudConvert\Models\Task;

    $obj_temp = $_FILES['file']['tmp_name'];
    $obj_name = $_FILES['file']['name'];
    $obj_errors = $_FILES['file']['error'];
    $obj_checksum = md5_file($obj_temp, false);
    $obj_size = filesize($obj_temp);
    $obj_extention = strtolower(pathinfo($obj_name, PATHINFO_EXTENSION));

    $cloudconvert = new CloudConvert([
        'api_key' => 'my_key_here',
        'sandbox' => false
    ]);

    $obj_key = $obj_checksum.'.'.$obj_extention;
    
    $job = (new Job())
    ->addTask(new Task('import/upload','upload-my-file'))
    ->addTask(
        (new Task('convert', 'convert-my-file'))
            ->set('input', 'upload-my-file')
            ->set('output_format', 'jpg')
    )
    ->addTask(
        (new Task('export/url', 'export-my-file'))
            ->set('input', 'convert-my-file')
    );

    $cloudconvert->jobs()->create($job);
    $url_task = $job->getTasks()->whereName('export-my-file')[0];
    $cloudconvert->tasks()->upload($url_task, fopen($obj_temp, 'r'), 'test.png');
    $cloudconvert->jobs()->wait($job);
    foreach ($job->getExportUrls() as $file) {
        echo $file->url;
    }

**Comment by mjwweb** [comment]
Update: 
I created a new Cloud Convert account and tried running the script again with a new API key. It went through without any issues. 

I tried switching back to my old API key from my original account and I'm getting the 'started too many jobs at once' error again. I even tried creating a new API key, but I'm still getting the same error response from the server.

I haven't used any conversion minutes in several days, and my dashboard still says I have 25 (I'm on the free tier). It seems there is something blocking requests from my account on your end.

Username for the account I'm having the issue with: Mikejw45
